yast/agents/smt_cron.scr

```diff
@@ -11,14 +11,14 @@
  *
```

Comment:
There is still `* SCR Agent for reading/writing /etc/smt.d/novell.com-smt` ^ line 5